docs/en/source/k8Introduction.rst

```diff
@@ -1,64 +1,37 @@
-****************************
-Introduction
-****************************
+********************************
+Scorpio Broker and Kubernetes
```

Comment:
[Minor] It should be "Scorpio Broker Kubernetes Deployment" instead of "Scorpio Broker and Kubernetes".

Reply:
Its a heading intentionally placed that way